amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1876,6 +1875,172 @@ VX_API_CALL vx_node VX_API_CALL  vxExtrppNode_SequenceRearrange(vx_graph graph,v
 	return node;
 }
 
+//tensor
+
+VX_API_ENTRY vx_node VX_API_CALL vxRppBrightness(vx_graph graph, vx_tensor pSrc, vx_tensor pSrcRoi, vx_tensor pDst, vx_array pAlpha, vx_array pBeta, vx_scalar inputLayout, vx_scalar outputLayout, vx_scalar roiType) {
```

Comment:
@rrawther @fiona-gladwin I am not sure if we want the tensor stuff in MIVisionX TOT. WE will only have these 5 then. I thought we were only doing bug fixes and image changes here. All the tensor stuff goes to rocAL repo. We will not add anything new to TOT here

Reply:
@fiona-gladwin - All tensor updates can be added to the rocAL repo.

Reply:
@LakshmiKumar23 @kiritigowda OpenVX extension support for RPP tensor augmentations has to be added here since rocal repo has only files wrt rocAL. 
@LakshmiKumar23 This will not break any of the existing rocAL functionalities since we do not call them from rocAL which is part of this repo. Please let us know if you want us to delete this PR for time being.

Reply:
@fiona-gladwin Oh sorry, I thought you added rocAL parts. Just rechecked, amd_rpp is okay